src/app/components/common/StatusBadge.tsx

```diff
@@ -57,14 +59,19 @@ const StyledBadge = styled(Box, {
     borderRadius: 10,
     padding: 4,
     paddingLeft: 10,
-    paddingRight: 5,
+    paddingRight: compact ? 5 : 10,
```

Comment:
Currently there is no need to use `compact`  prop as padding-right value will be always overwritten on mobile by a next CSS rule
```
    [theme.breakpoints.down('sm')]: {
      padding: 4,
    },
```